cloudify_azure/auth/oauth2.py

```diff
@@ -89,15 +91,15 @@ def request_access_token(self):
             'client_id': self.credentials.client_id,
             'client_secret': self.credentials.client_secret,
             'grant_type': constants.OAUTH2_GRANT_TYPE,
-            'resource': constants.OAUTH2_MGMT_RESOURCE
+            'resource': self.credentials.endpoint_resource
```

Comment:
Is this Azure Stack specific?

Reply:
Yes, AzureStack can use different url for auth.